NoteFind/View Controllers/NewNoteViewController.m

```diff
@@ -6,20 +6,16 @@
 //
 
 #import "NewNoteViewController.h"
-#import "TagsViewController.h"
-#import "Note.h"
-#import "TagCell.h"
-#import "TransferDelegate.h"
-#import "TextbookViewController.h"
-#import <MobileCoreServices/MobileCoreServices.h>
 
-
-@interface NewNoteViewController ()  <TransferDelegate, UIDocumentPickerDelegate>
+@interface NewNoteViewController ()  <TransferDelegate, UIDocumentPickerDelegate, UIImagePickerControllerDelegate, UINavigationControllerDelegate, ImportDelegate>
 @property (strong, nonatomic) NSMutableArray *tags;
```

Comment:
Lets add generics on the `tags` and `books` collections

Reply:
resolved!